resources/metadata/fortify.yaml

```diff
@@ -32,435 +32,419 @@ spec:
+        - GENERAL
+        - PARAMETERS
+        - STAGES
+        - STEPS
+      default: "/transfer/reportDownload.html"
     - name: pollingMinutes
       type: int
-      description: The number of minutes for which an uploaded FPR artifact's status is being polled to finish queuing/processing, if exceeded polling will be stopped and an error will be thrown
+      description: "The number of minutes for which an uploaded FPR artifact''s status is being polled to finish
+        queuing/processing, if exceeded polling will be stopped and an error will be thrown"
       scope:
-      - PARAMETERS
-      - STAGES
-      - STEPS
+        - PARAMETERS
+        - STAGES
+        - STEPS
       default: 30
     - name: quickScan
       type: bool
-      description: Whether a quick scan should be performed, please consult the related Fortify documentation on JAM on the impact of this setting
+      description: "Whether a quick scan should be performed, please consult the related Fortify documentation on
+        JAM on the impact of this setting"
       scope:
-      - PARAMETERS
-      - STAGES
-      - STEPS
+        - PARAMETERS
+        - STAGES
+        - STEPS
       default: false
     - name: translate
       type: string
-      description: "Options for translate phase of Fortify. Most likely, you do not need to set this parameter. See src, exclude. If `'src'` and `'exclude'` are set they are automatically used. Technical details: It has to be a JSON string of list of maps with required key `'src'`, and optional keys `'exclude'`, `'libDirs'`, `'aspnetcore'`, and `'dotNetCoreVersion'`"
+      description: "Options for translate phase of Fortify. Most likely, you do not need to set this parameter.
+        See src, exclude. If `'src'` and `'exclude'` are set they are automatically used. Technical details:
+        It has to be a JSON string of list of maps with required key `'src'`, and optional keys `'exclude'`,
+        `'libDirs'`, `'aspnetcore'`, and `'dotNetCoreVersion'`"
       scope:
         - PARAMETERS
         - STAGES
         - STEPS
     - name: src
-      type: string
-      description: Source directories to scan. Multiple entries are separated by space and wildcards can be used, e.g., `'src/main/resources/**/* src/main/java/**/*'`. If `translate` is set, this will ignored.
+      type: "[]string"
+      description: "A list of source directories to scan. Wildcards can be used, e.g., `'src/main/java/**/*'`.
+        If `'translate'` is set, this will ignored. The default value for `buildTool: 'maven'` is
+        ['**/*.xml', '**/*.html', '**/*.jsp', '**/*.js', '**/src/main/resources/**/*', '**/src/main/java/**/*'], for
+        `buildTool: 'pip'` it is ['./**/*']."
       scope:
         - PARAMETERS
         - STAGES
         - STEPS
-      default: "**/*.xml **/*.html **/*.jsp **/*.js **/src/main/resources/**/* **/src/main/java/**/*"
+      aliases:
+        - name: pythonIncludes
```

Comment:
As the "old" pythonIncludes is anyway not a list, I think we do not need the alias as it is anyway not compatible...

Reply:
Good point.